cms/djangoapps/contentstore/tests/test_signals.py

```diff
@@ -24,16 +28,15 @@ def setUp(self):
         CourseEnrollment.enroll(self.user, self.course.id)
 
     @patch('cms.djangoapps.contentstore.signals.handlers.cache.add')
-    @patch('cms.djangoapps.contentstore.signals.handlers.cache.delete')
     @patch('cms.djangoapps.contentstore.signals.handlers.task_compute_all_grades_for_course.apply_async')
     @ddt.data(True, False)
-    def test_locked(self, lock_available, compute_grades_async_mock, delete_mock, add_mock):
+    def test_locked(self, lock_available, compute_grades_async_mock, add_mock):
```

Comment:
Why you removed this param ?

Reply:
`delete_mock`

Reply:
this param wasn't being used in the test (unused parameter warning).